### PR TITLE
Add Hw9

### PR DESCRIPTION
**Pull request by olark1d**
Homework 9 is done, waiting for a feedback:)

### CODE REVIEW

File: HOMEWORK_9/EX2.CPP [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+
+void toUppercase(char str[])
+{
+    for (int i = 0; str[i] != '\0'; ++i) {
+        if (str[i] >= 'a' && str[i] <= 'z') {
+            str[i] = str[i] - 'a' + 'A';
```
Comment (bohdanpc) [inline HOMEWORK_9/EX2.CPP]: move this 'a' + 'A' into variable outside of the loop to not repeat this calculation every iteration of the loop

File: HOMEWORK_9/EX4.CPP [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    for(int i = 0; i < lenth; i++)
+    {
+        if(str[i] >= 'a' && str[i] <= 'z')
+        {
+            constonantsCount++;
+        }
+        else if(str[i] >= 'A' && str[i] <= 'Z')
+        {
+            vowelsCount++;       
+        }
```
Comment (bohdanpc) [inline HOMEWORK_9/EX4.CPP:16]: consonants and vowels, not lowers and Uppers 🥲

File: HOMEWORK_9/EX2.CPP (hunk 1 above)
Comment (olark1d) [reply HOMEWORK_9/EX2.CPP]: done

File: HOMEWORK_9/EX4.CPP (hunk 2 above)
Comment (olark1d) [reply HOMEWORK_9/EX4.CPP:16]: I think it looks better

File: HOMEWORK_9/EX4.CPP [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#include <cstring>
+
+void parseStringLetters(const char str[], int& vowelsCount, int& constonantsCount)
+{
+    int lenth = std::strlen(str);
+    for(int i = 0; i < lenth; i++)
+    {
+        if(str[i] >= 'a' && str[i] <= 'z')
+        {
+            constonantsCount++;
```
Comment (bohdanpc) [inline HOMEWORK_9/EX4.CPP:11]: consonantsCount

**Pull request merged**